src/elona/ui.cpp

```diff
@@ -612,6 +612,27 @@ void render_clock()
 
 
 
+void render_digital_clock()
```

Comment:
These text should be localized. Please add new entries in `runtime/locale/{jp,en}/ui.hcl` like this:

```hcl
digital_clock {
    time = "${_1}:${_2}:${_3}"
    date = "${_1}/${_2}/${3}"
}
```

In this case, there is no difference between English and Japanese, but all text should be localized for user extension with mods and another language.

Reply:
The locale resources are defined, but not used here.